corehq/apps/app_manager/management/commands/migrate_inline_search_instance.py

```diff
@@ -33,6 +33,8 @@ def migrate_app(self, app_doc):
                 should_save |= self._migrate_detail(app_doc, module, 'short')
                 should_save |= self._migrate_detail(app_doc, module, 'long')
 
+        if should_save and not app_doc.get('copy_of') and app_doc.get('version'):
```

Comment:
Would it make sense for `AppMigrationCommandBase` to handle this?

Reply:
Probably

Reply:
https://github.com/dimagi/commcare-hq/pull/31763

Reply:
🚀